lib/hooks/responses/index.js

```diff
@@ -212,9 +212,9 @@ function _mixin_jsonx(req, res) {
     sails.log.debug('***********************************************************************************');
 
     // Send conventional status message if no data was provided
-    // (see http://expressjs.com/api.html#res.send)
+    // (see http://expressjs.com/en/api.html#res.sendStatus)
     if (_.isUndefined(data)) {
-      return res.status(res.statusCode).send();
+      return res.sendStatus(res.statusCode);
```

Comment:
I think this change is incorrect; `res.status(code).send()` is perfectly valid; it sets the status code and sends an empty body. This change causes a body to be sent where it was not before, and the prior code isn't deprecated in any way.